src/test/authClass.loginWithMetamask.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai'
+import * as sinon from 'sinon'
+
+import { AuthService } from '../apiServices/AuthService'
+import { JwtTokensInterface } from '..'
+
+const postRequest = require('../http/index')
+
+describe('loginWithMetamask', () => {
+    const tokens: JwtTokensInterface = {
+        access: 'access',
+        refresh: 'refresh',
+    }
+
+    let postRequestStub: any
+    let setCredentialsStub: any
+
+    let AUTH
+    beforeEach(() => {
+        postRequestStub = sinon.stub(postRequest, 'postRequest')
+
+        AUTH = new AuthService('user', 'password')
+        setCredentialsStub = sinon.stub(AUTH, 'setCredentials')
```

Comment:
I don't think we need to stub setCredentials; we can just check if `AUTH.tokens` matches the tokens that we expect.